_blogposts/2020-12-07-release-8-4.mdx

```diff
@@ -55,30 +55,9 @@ dependent's command line flags for building binary artifacts. Such strategy bene
 
 ### Introducing `pinned-dependencies`
 
-To make `bsb -make-world` more practical, we also introduce a new concept called: `pinned-dependencies`.
-In general, packages are classified as three categories:
-
-- toplevel
-  - warnings reported
-  - warn-error respected
-  - build dev dependency
-  - run custom rules
-  - package-specs like ES6/CommonJS overrides all its dependencies
-- pinned dependencies
-  - warnings reported
-  - warn-error respected
-  - build dev dependency
-  - run custom rules
-- normal dependencies
-  - warnings, warn-error ignored
-  - ignore dev directories
-  - ignore custom generator rules
-
-Previously, we only had `toplevel` and `normal dependencies`, by introducing `pinned-dependencies`,
-such package will be built mostly like `toplevel` packages.
-
-The usage is quite simple, add `pinned-dependencies` in the toplevel package's `bsconfig.json`.
-Note such field only make sense in toplevel's configuration.
+To make `bsb -make-world` more practical for e.g. monorepo setups (`lerna`, `yarn workspaces`, etc.), we introduced a new concept called `pinned-dependencies`. A pinned dependency allows us to automatically rebuild packages that are pinned by a toplevel package, like your final webapp.
+
+Please refer to our [Monorepo Setup](/docs/manual/latest/build-monorepos) docs for more information on how to use `pinned-dependencies` with different monorepo setups.
```

Comment:
My understanding of monorepo is google or facebook style's monorepo which is one repo one version control system. I am not sure why people call this kind of setup monorepo, we do have good support for such monorepo since our build system is scalable.

For multiple packages development (pinned dependencies), our support is best effort, the user experience is not as good as "monorepo support" (as mentioned above), I am a bit worried about that when our users read such section, they assume we have very good support for their understanding of "monorepo". This might cause some user expectation mismatch

Reply:
Okay, so the term we are looking for is "Multiple Packages Development" instead of monorepo?
Yes, the term "monorepo" is a little convoluted in the JS world... I explicitly mentioned that the "original monorepo" use-case is well supported in the monorepo setup page